### ISSUE TITLE
[a11y-defect-3]: Use of deprecated "wizard" component results in H1 issues and increased cognitive load

### ISSUE THREAD

**Issue by coforma-jamie** [department-of-veterans-affairs/va.gov-team#63702]
## Point of contact

Jamie Klenetsky Fay (@coforma-jamie)

## Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

## Details

A [wizard pattern](https://design.va.gov/patterns/wizards), which is deprecated, is used to lead Vets to [Form 28-8832](https://staging.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-28-8832/introduction):
![Screenshot 2023-08-15 at 3 18 23 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/6eea975f-0c42-4d19-84ec-0cf4476d5ee4)

Using this pattern can be confusing for users. In this case, when navigating between the wizard version and intro version of the page, there's a very similar `<H1>`, and it may not be clear to screen reader users if they are actually navigating to separate pages.

## Reproduction steps

1. Go to [this page](https://staging.va.gov/careers-employment/education-and-career-counseling/apply-career-guidance-form-28-8832/introduction)
2. Answer the questions in the wizard (Veteran, No, Yes, Yes)
3. Click on "Apply for Personalized Career Planning and Guidance"
4. Scroll to the bottom and click "go back and answer the questions again"
5. This will take you back to the wizard, which has a very similar page title to the intro page.

### Short-term fix
You can use focus management to automatically focus on the `<H2>`, "Is this the form I need?" heading. That should make it more clear to an AT user what the page is for. Look at the [File a Supplemental Claim form](https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/start) to see it in action. When the page loads, focus automatically goes to the `<H2>`:
![Screenshot 2023-08-15 at 3 22 27 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/135633989/f5ad5d51-5b18-42ff-be95-a75aa1904f84)

### Long-term solution
As mentioned in the Design System [wizard pattern](https://design.va.gov/patterns/wizards) page, the [Wizard pattern is now deprecated](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/399) and replaced by the the [help users complete a sub-task pattern](https://design.va.gov/patterns/help-users-to/complete-a-sub-task). 

There are many accessibility defects related to the wizard pattern, as described [here](https://design.va.gov/patterns/wizards#retrofit-this-pattern).

We highly recommend that you use the sub-task pattern instead of the wizard pattern to avoid this accessibility issue and reduce the user's cognitive load.

## References, articles, or WCAG support

1. [Focus management example](https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/start)
2. Reduce cognitive load by having [one thing per page](https://www.gov.uk/service-manual/design/form-structure#start-with-one-thing-per-page)
3. [How to retrofit the wizard pattern, if you absolutely must keep it](https://design.va.gov/patterns/wizards#research-and-accessibility-rationale)

### Type of issue

- [X] Screenreader
- [ ] Keyboard
- [x] Focus
- [X] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else
```[tasklist]
### Tasks
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/67999
```

**Comment by sara-amanda** [comment]
Connected to Epic verified. Removing `Sitewide CAIA` label.
